frontend/src/contexts/unsavedChangesContext.ts

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react'
+
+import { UnsavedChangesHook } from '../../utils/hooks/useUnsavedChanges'
+
+const UnsavedChangesContext = createContext<UnsavedChangesHook>({
+  unsavedChanges: false,
+  setUnsavedChanges: () => null,
```

Comment:
Super nit-picky but a standard JS function that doesn't return anything would return `undefined` instead of `null`. `null` could suggest we are returning that value for a reason, but in this case we just want a function that does nothing.